app/views/posts/_form.html.erb

```diff
@@ -46,24 +47,27 @@
 </head>
 <body>
 
+<!-- precontent in the editor -->
```

Comment:
This looks good. It is very good practice to include some comments especially when several people are working on different part of a project like we are doing here!